api/meetups/serializers.py

```diff
@@ -18,3 +28,27 @@ class TagSerializer(serializers.ModelSerializer):
     class Meta:
         model = Tag
         fields = '__all__'
+
+
+class RsvpSerializer(serializers.ModelSerializer):
+    """
+    Serializers for rsvp
+    """
+    response = serializers.CharField()
+
+    def validate_response(self, value):
+        """
+        Validate User response
+        """
+
+        if "yes" not in value.lower():
+            if "no" not in value.lower():
+                if "maybe" not in value.lower():
+                    print(value)
```

Comment:
You forgot to remove the print statement

Reply:
Ahh thanks @a-braham , let me get rid of it